faq/security-self-assessment.md

```diff
@@ -0,0 +1,78 @@
+---
+description: This article describes how we use the Consensus Assessments Initiative Questionnaire (CAIQ) to assess our security posture and provide transparency for security control.
+---
+
+
```

Comment:
Prije tablice dodati onaj intro tekst kao što je na stranici octopusa, samo staviti naše info (syskit) unutra